samples/openthread/cli/harness/nRF_Connect_SDK.py

```diff
@@ -2837,6 +3141,15 @@ def __lstrip0x(s):
 
         return s
 
+    @staticmethod
+    def __normalize_pskd(strPSKd):
```

Comment:
nit. length could be checked/adjusted as well.
We expect 6-32 characters.

Reply:
I expect the changes in this commit to be reverted when DUA and Multicast support is added upstream. In any case the Test Harness should already pass a valid PSKd.